docs/guide/setup/defining_protocols.rst

```diff
@@ -3,46 +3,41 @@
 Creating and using Protocols
 ============================
 
-With a thermodynamic cycle of interest identified and represented using `ChemicalSystem` objects,
-the next step is to choose a method for estimating the free energy differences.
-This is done using a :class:`.Protocol`,
-a computational method for estimating the free energy difference between two chemical systems.
-
-Just as there are multiple possible methods for estimating free energy differences,
-there are multiple available Protocols to choose from.
-For example, included in the ``openfe`` package are the
-:class:`.RelativeHybridTopologyProtocol`,
-:class:`.AbsoluteSolvationProtocol`,
-and :class:`.PlainMDProtocol`;
-for a full list see :ref:`userguide_protocols`.
-This selection is being built upon,
-both by the openfe development team as well as with external academic groups.
-
-What all these Protocols share is a common interface for how they are created and executed,
-therefore it is relatively simple to
-try out a new method,
+A :class:`.Protocol` is a computational method for estimating the free energy difference between two chemical systems.
+
+Included in the ``openfe`` are the following common protocols:
```

Comment:
I believe we intentionally use the word "example" rather than just "these are common protocols", because we didn't want to go ahead and make them more valuable than any other protocol we might have.